scripts/library_handler.py

```diff
@@ -638,9 +640,16 @@ def _readDependencies(initFile):
       elif args.action == 'list':
         action = 'list'
         preamble = ''
+      if args.action == 'setup.cfg':
+        action = 'setup.cfg'
+        preamble = """#setup.cfg from library_handler.py
+[options]
+install_requires =
```

Comment:
change install_requires to installRequires?

Reply:
This is used by setuptools and cannot be changed.